pkg/systemd/services/timer-dialog.jsx

```diff
@@ -250,6 +252,11 @@ const CreateTimerDialogBody = ({ setIsOpen, owner }) => {
                             const min = repeatPatterns[idx].minute;
                             const validationFailedMinute = !(/^[0-9]+$/.test(min) && min <= 59 && min >= 0);
 
+                            if (repeat == 'monthly' && repeatPatterns.some(p => p.day > 28)) {
+                                validated = "warning";
+                                helperText = _("Choosing a day after the 28th limits the execution of the timer to the months which contain it");
```

Comment:
:thinking: that feels rather obvious to me.. did some user actually complain about this? 

Can we trim the string a bit, so that it's easier to translate and won't explode in length in other languages? How about "Will not run every month"?

Reply:
I also find it obvious and I am happy to drop this commit. I think it was just a question in the first SPUR.

Reply:
See the issue attached, there is a discussion about it.

Reply:
I removed the commit, I actually agree that it's not necessary.